src/libDirectoryService/Coinbase.cpp

```diff
@@ -137,8 +202,17 @@ void DirectoryService::InitCoinbase() {
   }
 
   LOG_MARKER();
+
+  VectorOfLookupNode vecLookup = m_mediator.m_lookup->GetLookupNodes();
+  const auto& epochNum = m_mediator.m_currentEpochNum;
+
   lock_guard<mutex> g(m_mutexCoinbaseRewardees);
 
+  for (const auto& lookupNode : vecLookup) {
+    m_coinbaseRewardees[epochNum][-2].push_back(
+        Account::GetAddressFromPublicKey(lookupNode.first));
+  }
+
   if (m_coinbaseRewardees.size() < NUM_FINAL_BLOCK_PER_POW) {
     LOG_GENERAL(INFO, "[CNBSE]"
                           << "Less then expected rewardees "
```

Comment:
I think we have to change these logs from "expected rewardees" to "expected tx epochs for rewarding" or something like that, and change INFO to WARNING. Also, probably good to actually log the size and NUM_FINAL_BLOCK_PER_POW.